src/jmcomic/api.py

```
@@ -78,31 +78,22 @@ def download_album(jm_album_id,
                    ) -> Union[__DOWNLOAD_API_RET, Set[__DOWNLOAD_API_RET]]:
     """
     下载一个本子（album），包含其所有的章节（photo）
-
-    当jm_album_id不是str或int时，视为批量下载，相当于调用 download_batch(download_album, jm_album_id, option, downloader)
-
+    当jm_album_id不是str或int时，视为批量下载
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)